src/gridfs/index.ts

```diff
@@ -144,11 +155,27 @@ export class GridFSBucket extends TypedEventEmitter<GridFSBucketEvents> {
    *
    * @param id - The id of the file doc
    */
-  async delete(id: ObjectId): Promise<void> {
-    const { deletedCount } = await this.s._filesCollection.deleteOne({ _id: id });
+  async delete(id: ObjectId, options?: { timeoutMS: number }): Promise<void> {
+    const { timeoutMS } = resolveOptions(this.s.db, options);
```

Comment:
I looked more closely at this - the constructor already resolves timeoutMS based on the `db` object and the bucket options.  These helpers should be resolving it based on the option set on the bucket and the options provided to each method - right?

```typescript
// constructor
const resolvedOptions = resolveOptions(this.db, { 
	...DEFAULT_GRIDFS_BUCKET_OPTIONS,
	...options,
})

// (write concern gets handled in resolveOptions)

// in each method
const timeoutMS = options.timeoutMS ?? this.s.options.timeoutMS;
```